.github/workflows/ci.yml

```diff
@@ -247,14 +252,6 @@ jobs:
           RUN: ${{ matrix.run }}
           RUNNERS: ${{ matrix.runners }}
         shell: bash
-
-      - name: Install cross
-        if: matrix.deploy
-        run: cargo install --path . --force
-
-      - name: Build cross-dev
-        run: cargo build --features=dev --bin cross-dev
-
```

Comment:
this is not needed anymore as we use cross-dev in ci now

Reply:
hmm, maybe it should be kept, just replaced with install https://github.com/cross-rs/cross/runs/6853748188?check_suite_focus=true